### PR TITLE
`Dh questions create update destroy 

### PR DESCRIPTION
**Pull request by DavidHoltkamp1**
# Description :: User Story

- Create, update, and destroy actions complete for questions. 
- Update and destroy functionality is living on the question show page, out of the user profile. 
- Recent Activity is on user profile, showing the last 3 questions asked by a particular user. 
- Destroying a question redirects back to the users profile currently. 
- Implement for_for forms for both create and edit a question. 

## Type of change

- [x] New Feature
- [ ] Bug Fix
- [ ] Refactor
- [ ] Breaking Change

## RSpec results

```
Finished in 4.66 seconds (files took 7.27 seconds to load)
56 examples, 0 failures

Coverage report generated for RSpec to /Users/davidholtkamp521/turing/3module/projects/Turing.SO/coverage. 152 / 229 LOC (66.38%) covered.
```

## Rubocop results

```
27 files inspected, 8 offenses detected
```

### CODE REVIEW

File: app/views/technical_forum/index.html.erb [hunk 1]
```diff
@@ -4,8 +4,7 @@
   <section id='question-<%= question.id %>'>
     <h4><%= link_to question.subject, "/questions/#{question.id}" %></h4>
     <p><%= question.content %></p>
-    <p><%= question.upvotes %></p>
+    <p>Upvotes: <%= question.upvotes %></p>
```
Comment (alerrian) [inline app/views/technical_forum/index.html.erb:7]: Nice refactor here.

File: app/views/users/profile/show.html.erb [hunk 2]
```diff
@@ -45,3 +45,13 @@
 <p><%= link_to "Edit", "/profile/edit" %> | <%= link_to "Delete", destroy_user_path(@user), method: :delete, data: { confirm: "You sure?" } %></p>
 
 <p><%= link_to "Change your password", edit_user_registration_path %></p>
+
+<h4>Your Recent Activity</h4>
```
Comment (alerrian) [inline app/views/users/profile/show.html.erb:49]: Cool. We'll tackle later/at another time exactly how to show this information to each user.

File: spec/features/users/questions/edit_spec.rb [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+      fill_in :question_subject, with: "Career advice"
+
+      click_on "Update Question"
+
+      expect(current_path).to eq("/questions/#{@question_2.id}")
+      expect(page).to have_content("Career advice")
+    end
+
+    it "I cannot leave a field blank when updating a question" do
```
Comment (alerrian) [inline spec/features/users/questions/edit_spec.rb:72]: Nice sad path here. I'll be able to build off of what we've got and continue sad path and feature testing what's left.

File: app/views/technical_forum/index.html.erb (hunk 1 above)
Comment (DavidHoltkamp1) [reply app/views/technical_forum/index.html.erb:7]: lol

**Pull request merged**